python/basic/Operators.C

```diff
@@ -74,8 +74,8 @@ struct OperatorInt {
 };
 
 struct OperatorLong {
-   operator long() { return m_long; }
-   long m_long;
+   operator intptr_t() { return m_long; }
```

Comment:
Do we want this change to be only done for Windows so that we still test long and ulong in the other platforms?

Reply:
Well, if you prefer to have #ifdefs in the code...

Reply:
Done